packages/material-ui/src/Button/Button.js

```diff
@@ -276,6 +276,7 @@ function Button(props) {
       disabled={disabled}
       focusRipple={!disableFocusRipple}
       focusVisibleClassName={classNames(classes.focusVisible, focusVisibleClassName)}
+      role="button"
```

Comment:
https://validator.w3.org/

![capture d ecran 2018-11-08 a 11 11 09](https://user-images.githubusercontent.com/3165635/48192051-07edac80-e347-11e8-9e36-0142ae38c95f.png)

Reply:
Can be conditionally applied for custom elements, maybe.

Reply:
Why not just apply it all the time thus providing a universal selector for testing? It's unnecessary not wrong. Adding logic is just another codepath which leads to bugs and bundle bloat.

Reply:
Should it be handled by the button base instead?

Reply:
Oh wait, it's already handled by the button base :)

Reply:
>  it all the time thus providing a universal selector for testing?

Why not targeting the button element for testing?

Reply:
Yes I write type="button"|role="button" in the tests. Sure.

Reply:
@eps1lon I see, it's not idea!